exercises/acronym/AcronymTest.fs

```diff
@@ -1,64 +1,65 @@
 module AcronymTest
 
 open NUnit.Framework
+open FsUnit
 open Acronym
 
 [<Test>]
 let ``Empty string abbreviated to empty string`` () =
-    Assert.That(acronym "", Is.EqualTo(""))
+    acronym "" |> should equal ""
```

Comment:
`should be NullOrEmptyString`

Reply:
Well, I don't think it should actually. The test case states explicitly that it should be abbreviated to an empty string. If I were to use `NullOrEmptyString`, returning `null` would also pass the test.

Reply:
Note: there also is a `EmptyString` variant, which I'll use.

Reply:
Sounds good!

Reply:
@ErikSchierboom did you decide not to update this to `EmptyString` for some reason?